app/assets/javascripts/discourse/app/components/sidebar/sidebar-section-form-link.hbs

```diff
@@ -0,0 +1,67 @@
+<div
```

Comment:
I don't think this file needs to be named `sidebar-section-form-link` and can just be `section-form-link` instead since it is already nested under the `sidebar` folder.